### ISSUE TITLE
V0.5.1 still not able to open websocket with auth turned on

### ISSUE THREAD

**Issue by sOm2y** [onmax/albatross-rpc-client-ts#59]
Hi team,
I noticed you recently pump up version to v 0.5.1 with a commit that enable websocket auth by attaching username and password as param query string.

#### here is my implementation using lastest package version

```
const url = new URL("http://validator-url:8648")
export const connectToRPC = (): NimiqRPCClient => {
    return new NimiqRPCClient(url , {

        username: "super" ,
        password: "secret"


    });
};
```
 ```
const { next, close } =
            await client.blockchainStreams.subscribeForLogsByAddressesAndTypes(
                {
                    addresses: [validatorWatchAddress],
                    types: [], 
                    withMetadata: true,
                },
            )
        next(async (data) => {

            let rawData:any = data
            console.log(JSON.stringify(rawData))
})
```

but i'm keep getting `{"error":{"code":1000,"message":"{}"}}` which i believe is 401 unauthroized, 
if i remove the auth object and turn off auth on my validator rpc server, the code works.

#### Also i tested in postman
this is the url gennerted from your pakcage which will not work in postman client 
`"ws://test.valiodator.com:8648/ws?username=super&password=secret"`   ----> 401

passing url as below on postman will be able to connect to rpc server
`ws://super:secret@test.valiodator.com:8648/ws `    ---> work

**Comment by onmax** [comment]
Could you try again installing the following version?

```
npm i https://pkg.pr.new/onmax/albatross-rpc-client-ts/nimiq-rpc-client-ts@69654be`
```

**Comment by sOm2y** [comment]
@onmax  getting 404 on this command, is this a private feed?

**Comment by onmax** [comment]
Sorry, I copied one more char.

Can you share with me the dependencies in `package.json`?

Instead of:

```
"albatross-rpc-client-ts": "0.5.1",
```

should be


```
"albatross-rpc-client-ts": "https://pkg.pr.new/onmax/albatross-rpc-client-ts/nimiq-rpc-client-ts@69654be",
```

Then run `npm install` (or `yarn install`/`pnpm install`) and then try

**Comment by sOm2y** [comment]
thanks @onmax, it works now

```
Initialising NimiqRPCClient...
Starting validator payout service...
{"data":{"data":{"type":"applied-block","inherents":[{"type":"payout-reward","to":"NQ95 CUFY Q8LC HC2H NQNM 6NM3 DT42 8Q72 N2BJ","value":409158}],"timestamp":1732741645815,"transactions":[]},"metadata":{"blockNumber":4166790,"blockHash":"7f39086750cd20bda9ea715386ab33940cb14c740c73658dc769d9ae59ebcec8"}},"metadata":{"blockNumber":4166790,"blockHash":"7f39086750cd20bda9ea715386ab33940cb14c740c73658dc769d9ae59ebcec8"}}
{"data":{"data":{"type":"applied-block","inherents":[{"type":"payout-reward","to":"NQ95 CUFY Q8LC HC2H NQNM 6NM3 DT42 8Q72 N2BJ","value":409428}],"timestamp":1732741704917,"transactions":[]},"metadata":{"blockNumber":4166850,"blockHash":"12c7b5dd65b40f6d955503142a5d734d1502dfcb70c3f3d974993f3b518113f5"}},"metadata":{"blockNumber":4166850,"blockHash":"12c7b5dd65b40f6d955503142a5d734d1502dfcb70c3f3d974993f3b518113f5"}}

```

**Comment by onmax** [comment]
Published `0.5.2`

**Comment by onmax** [comment]
you can now use "albatross-rpc-client-ts": "0.5.2"
